src/installer/view.rs

```diff
@@ -0,0 +1,1287 @@
+    ) -> Element<'a, Message> {
+        let mut col_address = Column::new()
+            .push(text::bold(text::simple("Address:")))
+            .push(
+                TextInput::new(&mut self.address_input, "Address", address, |msg| {
+                    Message::DefineBitcoind(message::DefineBitcoind::AddressEdited(msg))
+                })
+                .size(15)
+                .padding(10),
+            )
+            .spacing(10);
+        if warning_address {
+            col_address = col_address
+                .push(text::simple("Please enter correct address").color(color::WARNING));
+        }
+        let mut col_cookie = Column::new()
+            .push(text::bold(text::simple("Cookie path:")))
+            .push(
+                TextInput::new(
+                    &mut self.cookie_path_input,
+                    "Cookie path",
+                    cookie_path,
+                    |msg| Message::DefineBitcoind(message::DefineBitcoind::CookiePathEdited(msg)),
+                )
+                .size(15)
+                .padding(10),
+            )
+            .spacing(10);
+        if warning_cookie_path {
+            col_cookie =
+                col_cookie.push(text::simple("Please enter correct path").color(color::WARNING));
+        }
+        layout(
+            &mut self.scroll,
+            &mut self.previous_button,
+            Column::new()
+                .push(text::bold(text::simple("Define bitcoind")).size(50))
+                .push(Container::new(
+                    pick_list::PickList::new(
+                        &mut self.network_input,
+                        &NETWORKS[..],
+                        Some(*network),
+                        |msg| Message::DefineBitcoind(message::DefineBitcoind::NetworkEdited(msg)),
+                    )
+                    .padding(10),
+                ))
+                .push(col_address)
+                .push(col_cookie)
+                .push(
+                    button::primary(&mut self.save_button, button::button_content(None, "Save"))
+                        .on_press(Message::Next),
+                )
+                .width(Length::Fill)
+                .height(Length::Fill)
+                .padding(100)
+                .spacing(50)
+                .align_items(Align::Center)
+                .into(),
+        )
+    }
+}
+
+pub struct Final {
+    scroll: scrollable::State,
+    previous_button: Button,
+    action_button: Button,
+}
+
+impl Final {
+    pub fn new() -> Self {
+        Self {
+            scroll: scrollable::State::new(),
+            previous_button: Button::new(),
+            action_button: Button::new(),
+        }
+    }
+
+    pub fn render(
+        &mut self,
+        generating: bool,
+        config_path: Option<&std::path::PathBuf>,
+        warning: Option<&String>,
+    ) -> Element<Message> {
+        let mut col = Column::new()
+            .push(text::bold(text::simple("You reached the end")).size(50))
```

Comment:
I would put here "RevaultGUI is installed!" (or a better name, or a better phrasing) and the button for starting